client/my-sites/stats/stats-post-summary/index.jsx

```diff
@@ -191,7 +255,13 @@ class StatsPostSummary extends Component {
 					<QueryPostStats siteId={ siteId } postId={ postId } />
 
 					<StatsPeriodHeader>
-						<StatsPeriodNavigation showArrows={ false }>
+						<StatsPeriodNavigation
+							showArrows
+							onPeriodChange={ this.onPeriodChange }
+							disablePreviousArrow={ disablePreviousArrow }
+							disableNextArrow={ disableNextArrow }
+							date={ null }
```

Comment:
I will fix the [isToday check](https://github.com/Automattic/wp-calypso/blob/fc1f064037d0ee22facf4ab70a2fcf3864317b6b/client/my-sites/stats/stats-period-navigation/index.jsx#L246) in a follow-up PR, which would make passing `Null` date unnecessary.